src/modules/helpchan.ts

```diff
@@ -647,6 +654,18 @@ export class HelpChanModule extends Module {
 			.filter(it => it.topic !== helpChannelTopic);
 	}
 
+	calculateTimeout(lastMessageUnixTime: number) {
+		const timeMod = Math.floor(lastMessageUnixTime / 1000) % (60 * 60 * 24);
+
+		const apex = (9 + 5) * 60 * 60; // EST 9 am
+		const intercept = 10.125; // intercepts at 16 in the evening.
```

Comment:
My comment there is wrong, `intercept = 6.125` means the timeout is at its maximum at ±7 hours (16:00-02:00 EST).
`intercept = 10.125` means the timeout is at its maximum at ±9 hours from the apex (18:00-00:00 EST).